jboss-kie-smartrouter/added/launch/jboss-kie-smartrouter.sh

```diff
@@ -1,7 +1,9 @@
+
 #!/bin/bash
 
 source "${LAUNCH_DIR}/launch-common.sh"
 source "${LAUNCH_DIR}/logging.sh"
+source "${JBOSS_HOME}/bin/launch/jboss-kie-common.sh"
```

Comment:
For the JBOSS_HOME env var to be usable here, you will need to add it to the rhpam-7-image/smartrouter/modules/smartrouter/module.yaml file in the envs section, as that env var won't be available to you by default in non-EAP based image builds otherwise.